tests/returntypes/test_gpu_torch_new.py

```diff
@@ -0,0 +1,226 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
Amazing that this support is also being added! :star2:

Reply:
Going through the tests, I think it would be best to separate this file into its own PR as it seems to contain test cases that don't apply the interface, but rather:
* Perform device execution directly or;
* Use backprop.

(lmk if I missed something)

Otherwise, adding these checks may go "unnoticed" within the scope of the more major changes.